index.html

```diff
@@ -3,8 +3,38 @@
     <head>
         <meta name="viewport" content="width=device-width, initial-scale=1" charset="utf-8">
         <link rel="stylesheet" href="stetics.css"> 
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+        <link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" rel="stylesheet">
+        <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&display=swap" rel="stylesheet">
+        <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
     </head>
     <body>
-        <h1>Hello Microverse!</h1>
+        <h1 class="header">
+            <p class="logo">My Logo</p>
+            <div><img class="menu" src="Union.png" alt="MenuBar"></div>
+        </h1>
+        <div class="notheader">
+            <div class="textcontainer">
+                <div class="primarytext">Hey There.<br>
+                    I'm Gabriel.
+                </div>
+                <div class="supportingtext">
+                    I'm a Software Developer
```

Comment:
This is the second heading on the webpage, and it is important to use the proper tag for it which i believe is an `h2` tag. Using the `h2` will make the webpage easy for web crawlers to access and then to rank the page properly.